src/lext/include/lext/unique_name.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 Google LLC
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+#pragma once
+
+#include <string>
+
+namespace lext {
+class UniqueName
+{
+ public:
+  std::string GetUniqueName(const std::string& prefix = "lext_")
```

Comment:
It seems odd that prefix wouldn't be a ctor argument.  Would you really want to change prefixes per call?

Reply:
Again, it's in `rmp` now, I'd rather not touch it